openapi-spec/swagger.yaml

```diff
@@ -1124,11 +1124,6 @@ paths:
           in: body
```

Comment:
Honestly I don't like the empty body defined here, I would prefer to remove `body` object if nothing needs to be passed in the request body.